juju/model.py

```diff
@@ -1661,8 +1667,8 @@ async def _resolve_charm(self, url, origin):
             'charm-origin': {
                 'source': source,
                 'architecture': origin.architecture,
-                'track': origin.track,
-                'risk': origin.risk,
+                #'track': origin.track,
+                #'risk': origin.risk,
```

Comment:
Is this right? I'm sure we always want to pass this back.

Reply:
Some bad debugging code